### ISSUE TITLE
Error while running on Policy

### ISSUE THREAD

**Issue by StrAzure123** [rightscale/policy_sdk#17]
This is the I am geting while using .pt file for policy using fpt

```
.\fpt.exe -a run azure_subscription_access.pt param_azure_sub=$subscriptionId param_azure_tenant_id=$tenantId param_email=$emailAddress param_exclusion_tag_key="" param_azure_roles="Owner" --credentials= -k
```

removed Acct number and cred

```
[signal 0xc0000005 code=0x0 addr=0x10 pc=0xa43a08]

goroutine 1 [running]:
main.parseOptions(0xc00022a600, 0xc000094900, 0x5, 0x8, 0x1, 0x1, 0xbf51a6, 0x7, 0x0)
        C:/Users/travis/gopath/src/github.com/rightscale/policy_sdk/cmd/fpt/run.go:231 +0x848
main.policyTemplateRun(0xd07700, 0xc00001e018, 0xd150c0, 0xc00016c320, 0xc00001a400, 0x1c, 0xc000094900, 0x5, 0x8, 0xc000107530, ...)
        C:/Users/travis/gopath/src/github.com/rightscale/policy_sdk/cmd/fpt/run.go:56 +0x307
main.main()
        C:/Users/travis/gopath/src/github.com/rightscale/policy_sdk/cmd/fpt/main.go:142 +0xc62
Applying Policy with Subscription c478ebaa-4315-4eaf-95c8-273387418fd6 and Tenant: 4e9dbbfb-394a-4583-8810-53f81f819e3b Email: devanshu.prasad@stryker.com
Running azure_subscription_access.pt
Updated PolicyTemplate "Azure Subscription Access" (/api/governance/projects/121515/policy_templates/5e67feec6d456d00013d6609) from azure_subscription_access.pt
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x10 pc=0xa43a08]
```

**Comment by cdwilhelm** [comment]
@StrAzure123  thanks for reporting this.  We are reviewing the details and a possible solution.

**Comment by psschroeter** [comment]
Think is fixed now in 1.0.9 release. Issue was that a parameter was passed that didn't exist. It now correctly prints out the right error without the panic, thank for the bug report.
